static/src/javascripts/projects/commercial/modules/ad-verification/prepare-ad-verification.js

```diff
@@ -11,7 +10,7 @@ export const init = (start: () => void): Promise<void> => {
 
     start();
 
-    if (isInVariantSynchronous(commercialAdVerification, 'variant')) {
+    if (config.get('switches.confiant', true)) {
```

Comment:
Do we want this to be `true`?

This is a fallback value for if `switches` or `switches.confiant` is undefined.

Reply:
👍 I thought it was checking if the switch was on. Thanks for flagging. Will change to false.